.ci/.jenkins_nodejs.yml

```diff
@@ -0,0 +1,13 @@
+NODEJS_VERSION:
+  - "13"
+  - "13.0"
+  - "12"
+  - "12.0"
+  - "11"
+  - "11.0"
+  - "10"
+  - "10.0"
+  - "8"
+  - "8.6"
+  - "6"
```

Comment:
Node.js 6 really? EOL Jul 2019

![image](https://user-images.githubusercontent.com/5400788/71895980-c925d300-3152-11ea-95d1-db43daf1e6aa.png)

Reply:
That's actually the list of versions that were declared in the .travis.yml, so no issues to remove it, but let's tackle that in a follow up if required :)